packages/token-search-discovery-controller/src/token-discovery-api-service/token-discovery-api-service.ts

```diff
@@ -39,4 +44,60 @@ export class TokenDiscoveryApiService extends AbstractTokenDiscoveryApiService {
 
     return response.json();
   }
+
+  async getTopLosersByChains(
+    topLosersParams: TopLosersParams,
+  ): Promise<MoralisTokenResponseItem[]> {
+    const url = new URL('/tokens-search/top-losers-by-chains', this.#baseUrl);
+
+    if (topLosersParams.chains && topLosersParams.chains.length > 0) {
+      url.searchParams.append('chains', topLosersParams.chains.join());
+    }
+    if (topLosersParams.limit) {
+      url.searchParams.append('limit', topLosersParams.limit);
+    }
+
+    const response = await fetch(url, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
```

Comment:
We could consider creating a common fetcher function eventually

Reply:
100% agree, here's a fast follow ticket: https://consensyssoftware.atlassian.net/browse/MMPD-1552